app/src/main/java/org/fossasia/openevent/app/utils/CompareUtils.java

```diff
@@ -0,0 +1,24 @@
+package org.fossasia.openevent.app.utils;
+
+import io.reactivex.functions.Function;
+import timber.log.Timber;
+
+public class CompareUtils {
+
+    @SafeVarargs
+    public static <T> int compareCascading(T one, T two, Function<T, String>... mappers) {
+        for (Function<T, String> mapper : mappers) {
+            try {
+                int current = mapper.apply(one).toLowerCase().compareTo(mapper.apply(two).toLowerCase());
+                if (current != 0)
+                    return current;
+            } catch (Exception e) {
+                // No-Op Can't happen
+                Timber.wtf(e);
+            }
+        }
+
+        return 0;
```

Comment:
yeah. but we don't want that. As it is in `compareTo` interface. This shall give equality for two different attendees if firstname, lastname and emails are matched in lowercase. For equality we should consider attendee's id only right? so here only either 1 or -1 should be returned.